### DISCUSSION TITLE
2023/01/rumus-excel-menampilkan-data-keuangan-tertentu

### DISCUSSION THREAD

**Discussion by giscus** [madi10/MANTANKODE#33] [General]
# 2023/01/rumus-excel-menampilkan-data-keuangan-tertentu

Menampilkan Jumlah Keuangan dengan Kata Tertentu di Excel

https://www.mantankode.net/2023/01/rumus-excel-menampilkan-data-keuangan-tertentu.html?m=1

**Comment by madi10** [comment]
Pass: mantankode.net
